src/App.js

```diff
@@ -79,15 +78,19 @@ function App() {
                 />
             )}
             {setsSuccess && typesSuccess && (
-                <AddPokemon types={types} sets={sets} />
+                <AddPokemon
+                    types={types}
+                    sets={sets}
+                    updateCardData={updateFormData}
+                />
             )}
             {selectedPokemon !== 0 && setsSuccess && typesSuccess && (
                 <EditPokemon
                     sets={sets}
                     setSelectedPokemon={setSelectedPokemon}
                     selectedPokemon={selectedPokemon}
                     types={types}
-                    updateCardData={updateCardData}
+                    updateCardData={updateFormData}
                     cardData={pokemonCardData}
```

Comment:
I think since this is only used in Edit Pokemon now that we can move it into there and not decalre it in App

Reply:
nvm it looks like we need this because of it being set in Table when a pokemon is selected